src/com/amazon/ion/benchmark/GenerateAndOrganizeBenchmarkResults.java

```diff
@@ -62,10 +82,28 @@ public static void generateAndSaveBenchMarkResults(Map<String, Object> optionsMa
      * @throws IOException if error occurs when reading executing process.
      */
     private static void printExecuteProcess(Process process) throws IOException {
-        BufferedReader reader = new BufferedReader(new InputStreamReader(process.getInputStream()));
-        String line;
-        while ((line = reader.readLine()) != null) {
-            System.out.println(line);
+        try (BufferedReader reader = new BufferedReader(new InputStreamReader(process.getInputStream()))) {
+            String line;
+            while ((line = reader.readLine()) != null) {
+                System.out.println(line);
+            }
+            System.out.println(System.lineSeparator());
         }
     }
+
+    /**
+     * Parse pom.xml and extract the version of ion-java-benchmark-cli.
+     * @param filePath the file path of pom.xml/
+     * @return the version of ion-java-benchmark-cli in a string format.
+     * @throws Exception if errors occur when parsing data.
+     */
+    private static String pomParser(String filePath) throws Exception {
```

Comment:
This method should be named something like "parseVersionFromPom".

Reply:
> This method should be named something like "parseVersionFromPom".

Sounds good. Change will be updated.